net-wireless/uhd/uhd-4.1.0.5.ebuild

```diff
@@ -32,14 +31,15 @@ RDEPEND="${PYTHON_DEPS}
 	e300? ( virtual/udev )
 	usb? ( virtual/libusb:1 )
 	dev-libs/boost:=
```

Comment:
>= on 1.65

Reply:
last boost <1.65 was removed in Apr 2019, so safe to leave this off
https://github.com/gentoo/gentoo/commit/87700b05371354405a1cbb7b9a4744fb8ff3bfb9